server/lib/plugins/push.js

```diff
@@ -63,22 +58,11 @@ async function pushPlugin(server, options) {
   const expo = new Expo()
 
   const push = {
-    send: async ({ subscriptions, secretId, uniqueId, requestObj }) => {
-      return Promise.all(
-        subscriptions.map(async (doc) => {
-          const subscription = doc.data()
-
-          if (subscription.type === 'expo') {
-            return sendExpoPush(log, expo, { subscription, secretId, uniqueId })
-          }
-
-          return sendWebPush(log, requestObj, {
-            subscription,
-            secretId,
-            uniqueId
-          })
-        })
-      )
+    send: async ({ subscription, secretId, uniqueId }) => {
+      const params = { subscription, secretId, uniqueId }
+      subscription.type === 'expo'
+        ? sendExpoPush(log, expo, params)
+        : sendWebPush(log, params)
```

Comment:
why this change?

Reply:
Earlier this method received multiple subscriptions which is why it used `map`. Now this method is called with only one subscription so I have simplified the code